client/blocks/simple-site-rename-form/dialog.jsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal Dependencies
+ */
+import Dialog from 'components/dialog';
+import FormLabel from 'components/forms/form-label';
+import FormInputCheckbox from 'components/forms/form-checkbox';
+
+class SiteRenamerConfirmationDialog extends PureComponent {
+	static propTypes = {
+		currentDomainSuffix: PropTypes.string,
+		isVisible: PropTypes.bool,
+		newDomainSuffix: PropTypes.string,
+		onConfirm: PropTypes.func.isRequired,
+		onClose: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		onClose: noop,
+		onConfirm: noop,
+		currentDomainSuffix: '.wordpress.com',
+		newDomainSuffix: '.wordpress.com',
+	};
+
+	state = {
+		isConfirmationChecked: false,
+	};
+
+	toggleConfirmationChecked = () => {
+		this.setState( {
+			isConfirmationChecked: ! this.state.isConfirmationChecked,
+		} );
+	};
+
+	onConfirm = closeDialog => {
+		this.onClose();
+		this.props.onConfirm( this.props.targetSite, closeDialog );
+	};
+
+	onClose = () => {
+		this.props.onClose();
+		this.setState( {
+			isConfirmationChecked: false,
+		} );
+	};
+
+	render() {
+		const {
+			currentDomainName,
+			currentDomainSuffix,
+			isVisible,
+			newDomainName,
+			newDomainSuffix,
+			translate,
+		} = this.props;
+		const buttons = [
+			{
+				action: 'cancel',
+				label: translate( 'Cancel' ),
+			},
+			{
+				action: 'confirm',
+				label: translate( 'Change Site Address' ),
+				onClick: this.onConfirm,
+				disabled: ! this.state.isConfirmationChecked,
+				isPrimary: true,
+			},
+		];
+
+		return (
+			<Dialog
+				className="simple-site-rename-form__dialog"
+				isVisible={ isVisible }
+				buttons={ buttons }
+				onClose={ this.onClose }
+			>
+				<h1>{ translate( "Let's Review" ) }</h1>
+				<p>
+					{ translate(
+						'You are about to change your domain name. Once changed, ' +
+							'your previous domain name will be unavailable for you or anyone else.'
+					) }
+				</p>
+				<div className="simple-site-rename-form__confirmation-detail">
+					<Gridicon icon="cross-circle" size={ 18 } className="simple-site-rename-form__copy-red" />
```

Comment:
`simple-site-rename-form__copy-red`
Are we using this explicit pattern anywhere else? Wondering if we'd want something like e.g. `copy-alert` and `copy-success` instead of `copy-red` and `copy-green`. If at some point we're rebranding or theming, our colors might change and we'd have to change the classes, too.

Reply:
Good call!

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/22671